word-search-page/word-search.js

```diff
@@ -0,0 +1,36 @@
+//Selectors
+function get(id1) {
+    return document.getElementById(id1);
+}
+let inputValue = get("search-input");
+let searchButton = get("search-button");
+let searchResult = get("search-result");
+let addResult = document.createElement('li');
+let word = inputValue.value;
+var verse;
+searchButton.addEventListener("click", search );
+
+
+function search(){
+    let url = "https://unpkg.com/quran-json@1.0.1/json/quran/en.json";
+    fetch(url)
+
+    .then(response=>{
+        findConsole = response.json();
+        console.log(findConsole);
+        return findConsole  ;
+    })
+    
+    .then(findConsole =>{
+            for ( let i = 0; i< findConsole.length; i++ ){ 
+                console.log(word);
+                console.log(findConsole[i].text)
+            if ( word ===  findConsole[i].text.includes(word)) {
```

Comment:
- you have to get the input value here, : `word.value` not at the beginning. 
- Fix the condition , here you want to check if the word is a part of the text or not ,so the result will be true or false which is achieved by this condition  :  `findConsole[i].text.includes(word)` only so, the first part of the condition is wrong.